config/locales/en.yml

```diff
@@ -1393,6 +1393,7 @@ en:
       reg_opens_too_early: "Registration is set to start in less than the required 48 hours."
       registration_already_closed: "The deadline to register has already passed. Please set it to be in the future."
       tooltip_registered: "You are registered."
+      tooltip_pending: "Your registration is awaiting manual approval. Please ensure you have followed any payment instructions in the Competition Page."
```

Comment:
Not only **payment** instructions -- it could really be _any_ instructions (for example, "Don't forget to enter a venue name in the comment field or else we cannot approve you")

Reply:
I'm just a bit worried about cluttering this field - and I'm assuming that if a user has paid  but not fulfilled some other requirement, organizers will reach out to the competitor for the missing information

Reply:
Perhaps: 
> Please ensure you have followed any instructions (eg, payment) listed in the Competition Page